### PR TITLE
Minor cleanup in bubblebench.ts

### PR DESCRIPTION
**Pull request by DLehenbauer**
Spotted a couple minor opportunities for cleanup while copying code to bootstrap a SharedTree sample.

### CODE REVIEW

File: experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts [hunk 1]
```diff
@@ -14,30 +14,24 @@ export class Bubblebench extends DataObject {
 	}
 	private maybeTree?: ISharedTree = undefined;
 	private maybeAppState?: AppState = undefined;
-	static treeFactory = new SharedTreeFactory();
```
Comment (DLehenbauer) [inline experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts:17]: There is no need to cache the `SharedTreeFactory` instance.

For comparison, see the implementations of `SharedFoo.getFactory()`, which returns `new SharedFooFactory()` (uncached).

File: experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts [hunk 2]
```diff
@@ -14,30 +14,24 @@ export class Bubblebench extends DataObject {
...
 		this.maybeTree = this.runtime.createChannel(
-			"unique-bubblebench-key-1337",
-			Bubblebench.treeFactory.type,
+			/* id: */ undefined,
+			new SharedTreeFactory().type,
 		) as ISharedTree;
 
 		this.initializeTree(this.maybeTree);
 
-		this.root.set("unique-bubblebench-key-1337", this.maybeTree.handle);
```
Comment (DLehenbauer) [inline experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts:27]: There is only a single key inserted for the lifetime of this document, which occurs here at construction.

File: experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts [hunk 3]
```diff
@@ -14,30 +14,24 @@ export class Bubblebench extends DataObject {
...
-			.get<IFluidHandle<ISharedTree>>("unique-bubblebench-key-1337")!
-			.get();
+		this.maybeTree = await this.root.get<IFluidHandle<ISharedTree>>("tree")!.get();
 	}
 
 	protected async hasInitialized() {
-		if (this.tree === undefined) {
-			throw new Error("hasInitialized called but tree is still undefined");
-		}
 		this.maybeAppState = new AppState(
```
Comment (DLehenbauer) [inline experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts:35]: `this.tree` already throws if not yet initialized.

File: experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts [hunk 4]
```diff
@@ -14,30 +14,24 @@ export class Bubblebench extends DataObject {
...
 
-		this.root.set("unique-bubblebench-key-1337", this.maybeTree.handle);
+		this.root.set("tree", this.maybeTree.handle);
 	}
 
 	protected async initializingFromExisting() {
 		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-		this.maybeTree = await this.root
-			.get<IFluidHandle<ISharedTree>>("unique-bubblebench-key-1337")!
-			.get();
```
Comment (DLehenbauer) [inline experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts:34]: Prettier

File: experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts (hunk 2 above)
Comment (ChumpChief) [reply experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts:27]: In other examples we pull this ID out into a const above for reuse in initializingFromExisting.

File: experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts [hunk 5]
```diff
@@ -14,30 +14,24 @@ export class Bubblebench extends DataObject {
 	}
 	private maybeTree?: ISharedTree = undefined;
 	private maybeAppState?: AppState = undefined;
```
Comment (ChumpChief) [inline experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts:16]: My personal preference is to type these as non-optional but unioned with undefined as I think it communicates the two-state nature better?  But might just be an opinion.

```suggestion
	private _tree: ISharedTree | undefined;
	private _appState: AppState | undefined;
```

**Review by ChumpChief — APPROVED** [review]

**Comment by DLehenbauer** [comment]
Thanks, @ChumpChief.  All good suggestions!

**Pull request merged**